src/middlewared/middlewared/etc_files/local/nslcd.conf

```diff
@@ -18,6 +20,15 @@
             ldap = None
 
         ldap_enabled = ldap['enable']
+        for param in ldap['auxiliary_parameters'].splitlines():
+            param = param.strip()
+            if not param.startswith('nss_min_uid'):
+                aux.append(param)
+            else:
+                try:
+                    min_uid = param.split()[1]
```

Comment:
How do the params look like?

`nss_min_uid = 4000`

is possible? If so you would get "=" here, and not 4000

Reply:
"nss_min_uid 4000" No equals sign.